### PR TITLE
fix(simple-list): added tabindex, button type, removed hover underline

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/2674

### CODE REVIEW

File: src/patternfly/components/SimpleList/simple-list-item-link.hbs [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <{{#if simple-list-item-link--type}}{{simple-list-item-link--type}}{{else}}button{{/if}} class="pf-c-simple-list__item-link{{#if simple-list-item-link--modifier}} {{simple-list-item-link--modifier}}{{/if}}"
```
Comment (mattnolting) [inline src/patternfly/components/SimpleList/simple-list-item-link.hbs]:
```suggestion
<{{#if simple-list-item-link--IsLink}}a{{else}}button{{/if}} class="pf-c-simple-list__item-link{{#if simple-list-item-link--modifier}} {{simple-list-item-link--modifier}}{{/if}}"
```

File: src/patternfly/components/SimpleList/simple-list-item-link.hbs [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+  {{#if simple-list-item-link--type}}
```
Comment (mattnolting) [inline src/patternfly/components/SimpleList/simple-list-item-link.hbs]:
```suggestion
  {{#if simple-list-item-link--IsLink}}
```

File: src/patternfly/components/SimpleList/examples/SimpleList.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ cssPrefix: pf-c-simple-list
       {{/simple-list-item-link}}
     {{/simple-list-item}}
     {{#> simple-list-item}}
-      {{#> simple-list-item-link}}
+      {{#> simple-list-item-link simple-list-item-link--type="a"}}
```
Comment (mattnolting) [inline src/patternfly/components/SimpleList/examples/SimpleList.md]:
```suggestion
      {{#> simple-list-item-link simple-list-item-link--IsLink="true"}}
```

File: src/patternfly/components/SimpleList/examples/SimpleList.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ cssPrefix: pf-c-simple-list
     {{/simple-list-title}}
     {{#> simple-list-list}}
       {{#> simple-list-item}}
-        {{#> simple-list-item-link simple-list-item-link--modifier="pf-m-current"}}
+        {{#> simple-list-item-link simple-list-item-link--type="a" simple-list-item-link--modifier="pf-m-current"}}
```
Comment (mattnolting) [inline src/patternfly/components/SimpleList/examples/SimpleList.md]:
```suggestion
        {{#> simple-list-item-link simple-list-item-link--IsLink="true" simple-list-item-link--modifier="pf-m-current"}}
```

File: src/patternfly/components/SimpleList/examples/SimpleList.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ cssPrefix: pf-c-simple-list
         {{/simple-list-item-link}}
       {{/simple-list-item}}
       {{#> simple-list-item}}
-        {{#> simple-list-item-link}}
+        {{#> simple-list-item-link simple-list-item-link--type="a"}}
```
Comment (mattnolting) [inline src/patternfly/components/SimpleList/examples/SimpleList.md]:
```suggestion
        {{#> simple-list-item-link simple-list-item-link--IsLink="true"}}
```

File: src/patternfly/components/SimpleList/examples/SimpleList.md [hunk 6]
```diff
@@ -71,7 +71,7 @@ cssPrefix: pf-c-simple-list
...
```
Comment (mattnolting) [inline src/patternfly/components/SimpleList/examples/SimpleList.md]:
```suggestion
        {{#> simple-list-item-link simple-list-item-link--IsLInk="true"}}
```

**Comment by christiemolloy** [comment]
I find that having the link in the same list as the buttons, can be a weird interaction for the user because the focus ring changes so abruptly on the link. Could you add another example with just links and split the buttons from the links. Or could you change the text that sits inside them to be List item - button or list item - link?

**Comment by mcoker** [comment]
Yeah that's a good idea. I just matched what was in react, but I think you'd either have a list with buttons or links - not both. Will update and ask for the same in the react example.

**Pull request merged**